mosec/dry_run.py

```diff
@@ -133,12 +133,26 @@ def run(self):
 
         logger.info("dry run init successful")
         self.warmup()
+
+        logger.info("wait for worker init done")
+        if not self.event.is_set():
+            self.event.set()
+        for i, process in enumerate(self.pool):
+            process.join()
```

Comment:
If some process has error some doesn't, will this be blocked?

Reply:
It will block if the current process never finishes the init. But if some processes have exceptions, the processes will return immediately. Since everyone already received the stop event, the normal processes will exit after initing.